test/unit/gpu/Traversal.cxx

```diff
@@ -236,6 +236,30 @@ int main(int argc, char *argv[])
     s_ntests_passed++;
   }
 
+#if defined(RAJA_ENABLE_AGENCY) && 0
```

Comment:
This was a preliminary addition of tests for the CUDA side, however it doesn't yet compile (for unrelated reasons) thus we can leave it alone for now.